### PR TITLE
feat: Parse mentions from message input (WEBAPP-5443)

### PR DESCRIPTION
**Pull request by AndyLnd**
This is just the method for parsing the mentions from the input. Nothing happens with that for now.

### CODE REVIEW

**Comment by atomrc** [comment]
Ah no, it's not a pure function, but definitely something we want to have tests for

**Comment by AndyLnd** [comment]
I will make it pure and add tests.

File: app/script/view_model/content/InputBarViewModel.js [hunk 1]
```diff
@@ -356,10 +356,31 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
...
   sendMessage(messageText) {
     if (messageText.length) {
+      const mentions = this.parseForMentions(messageText, this.conversationEntity().participating_user_ets());
+      this.logger.info(`Found ${mentions.length} mentions in outgoing message.`);
       this.conversationRepository.send_text_with_link_preview(messageText, this.conversationEntity());
     }
   }
 
+  parseForMentions(messageText, userEntities) {
+    const mentionRegexp = /\B@(.+?)\b/g;
```
Comment (atomrc) [inline app/script/view_model/content/InputBarViewModel.js:366]: Are you sure about the `\B`. Shouldn't it be `\b` (word boundary)?

**Comment by gregor** [comment]
LGTM

**Pull request merged**